From f58d716ba8a459d07934cb514ce6045dc8e0b477 Mon Sep 17 00:00:00 2001
From: Bidipta Samadder <bidiptax.samadder@intel.com>
Date: Wed, 13 May 2015 12:15:33 +0530
Subject: [PATCH] pmic_ccsm: Fix pmic ccsm suspend/resume logic

This patch changes the late suspend and early resume
calls to regular suspend and resume calls to avoid early
pmic access.

Change-Id: Ie7072d5a82d4435af9b5acf0e54d5db4c598fe2d
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1591
Signed-off-by: Bidipta Samadder <bidiptax.samadder@intel.com>
Signed-off-by: Ramakrishna Pallala <ramakrishna.pallala@intel.com>
---
 drivers/power/intel_pmic_ccsm.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/power/intel_pmic_ccsm.c b/drivers/power/intel_pmic_ccsm.c
index 1c823702b65a..a432489fa1a0 100644
--- a/drivers/power/intel_pmic_ccsm.c
+++ b/drivers/power/intel_pmic_ccsm.c
@@ -447,8 +447,8 @@ static int pmic_ccsm_resume(struct device *dev)
 }
 
 const struct dev_pm_ops pmic_ccsm_pm = {
-	.suspend_late = pmic_ccsm_suspend,
-	.resume_early = pmic_ccsm_resume,
+	.suspend = pmic_ccsm_suspend,
+	.resume = pmic_ccsm_resume,
 };
 
 static const struct file_operations pmic_chrgr_tt_reg_fops = {
-- 
1.9.1

